website/static/js/projectSettings.js

```diff
@@ -38,19 +44,38 @@ var NodeCategorySettings = oop.extend(
 
             self.selectedCategory = ko.observable(category);
             self.dirty = ko.observable(false);
+
+            self.dirtyTitleDescription = ko.computed(function(){
+                if (self.title() !== self.decodedTitle || 
+                    self.description() !== self.decodedDescription) {
+                    return true;
+                }
+                else {
+                    return false;
+                }
+                }, self);
+            
             self.selectedCategory.subscribe(function(value) {
                 if (value !== self.category()) {
                     self.dirty(true);
                 }
-            });
+            });    
         },
-        updateSuccess: function(newcategory) {
+
+        /*success handlers*/
+        updateCategorySuccess: function(newcategory) {
             var self = this;
             self.changeMessage(self.UPDATE_SUCCESS_MESSAGE, self.MESSAGE_SUCCESS_CLASS);
             self.category(newcategory);
             self.dirty(false);
         },
-        updateError: function(xhr, status, error) {
+        //TODO - refactor title/description success handler to display success message 
+        updateTitleDescriptionSuccess: function() {
+            window.location.reload();   
```

Comment:
After further review, I'm OK with reloading the page. It's not perfect, but updating the title outside of Knockout or leaving it displayed incorrectly are both greater evils :)